docs/api/web-contents.md

```diff
@@ -1081,6 +1114,17 @@ Works like `executeJavaScript` but evaluates `scripts` in an isolated context.
 
 Ignore application menu shortcuts while this web contents is focused.
 
+#### `contents.setWindowOpenOverride(handler)`
+
+* `handler` Function
+  * `details` Object
+    * `url` String - URL passed to `window.open()`
```

Comment:
Can we specify, is this the _resolved_ URL or the string that was passed to that method. i.e. does `windoe.open('/foo')` result in `http://example.com/foo` or just `/foo`

Reply:
Good point — it is the resolved URL. Will add a note.